core/quantity/README.md

```diff
@@ -2,10 +2,69 @@
 
 Copyright © Bentley Systems, Incorporated. All rights reserved. See LICENSE.md for license terms and full copyright notice.
 
-## Description
-
 The __@itwin/core-quantity__ package contains classes for quantity formatting and parsing.
 
-## Documentation
+## Terms and Concepts
+
+### Common Terms
+
+- [Unit]($quantity)/[UnitProps]($quantity) - A named unit of measure which can be located by its name or label. The definition of any unit is represented through it's `UnitProps`.
+- [UnitsProvider]($quantity) - A class that will also locate the UnitProps for a unit given name or label. This class will also provide a [UnitConversion]($quantity) to convert from one unit to another.
+- Unit Family/[Phenomenon]($ecschema-metadata) - The physical quantity that this unit measures (e.g., length, temperature, pressure).  Only units in the same phenomenon can be converted between.
+- Persistence Unit - The unit used to store the quantity value in memory or to persist the value in an editable IModel. While there is **NO** explicit flag or property to denote a persistence unit, those units are typically a part of the `Units.SI` [UnitSystem]($ecschema-metadata). The unit conversion of any persistence unit should both have its numerator and denominator set to 1.
```

Comment:
This could be confusing: 'any persistence unit should both have it's numerator and denominator set to 1', I'm pretty sure that's true but so many other units have this true that I'm not sure it matters.  

Also this is mixing levels.  Persistence Unit is the unit used to persist the value, what we choose as the persistence unit is based on our schema standards.  There is nothing in core quantity that mandates that persistence unit be SI.

Reply:
okay, so maybe I'll put the blurb about persistence unit in the learning doc in core-frontend?

Reply:
QuantityFormatting.md already had a sentence on persistence unit from Bill, so I switched to using that